src/rhsm/profile.py

```diff
@@ -72,6 +72,12 @@ def __eq__(self, other):
     def __str__(self):
         return "<Package: %s %s %s>" % (self.name, self.version, self.release)
 
+    @staticmethod
+    def _normalize_string(value):
+        if value is None:
+            return None
+        return value.decode('utf-8', errors='replace')
```

Comment:
What about following case: Python 3 is used and type of value is `str` (`dnf` is used instead of `yum`). The value will not have method `decode()` in this case.

Reply:
I did a check to see what the datatype returned was and even in Python 3 it was `bytes`; however, I think it's a good idea to handle the `str` case in order to defend against possible changes in behavior. Will update the PR.

Reply:
Yeah, my concerns were about possible changes in the future.